### ISSUE TITLE
applet UIDs incorrectly set to False

### ISSUE THREAD

**Issue by dkienzler** [m-labs/artiq#430]
Hi, 

I'm a new postdoc at NIST.

Hope this wasn't already reported.

Terminal output:
Traceback (most recent call last):
  File "C:\Anaconda3\envs\artiq-main\lib\site-packages\artiq\gui\applets.py", line 328, in delete
    self.applet_uids.remove(item.applet_uid)
KeyError: False

Best,
Daniel

**Comment by sbourdeauducq** [comment]
The condition is not "the last applet", but I've been able to reproduce this on Linux.

**Comment by sbourdeauducq** [comment]
though only partially. I had several applet UIDs incorrect set to ``False`` in my ``artiq_dashboard.pyon`` which is what was causing this bug, but I don't know how they ended up there.

**Comment by dkienzler** [comment]
Thanks.

D.

**Comment by jordens** [comment]
I see this as well with the browser.

```
    "AppletsDock": [(1, true, "", "$python -m artiq.applets.plot_xy --embed $ipc_address --x flopping_f_frequency flopping_f_brightness", null), (false, true, "", "$python -m artiq.applets.plot_xy --embed $ipc_address --x flopping_f_frequency flopping_f_brightness", null)]
}
```

**Comment by jordens** [comment]
From #444
Delete artiq_gui.pyon and start the GUI
Right click in applets pane -> new applet
Right click in applets pane -> new applet.

**Comment by jordens** [comment]
fixed in 109aa73
